tasks/main.yml

```diff
@@ -1,6 +1,23 @@
 # tasks/main.yml
```

Comment:
Decide please if you use old-style module names or [fully-qualified collection](https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_2.10.html) names

(I mean across all repository not only here)